tests/devices/unit_tests/electron_analyser/util.py

```diff
@@ -79,7 +79,14 @@ def assert_region_has_expected_values(
 ) -> None:
     for key in r.__dict__:
         if key in expected_region_values:
-            assert r.__dict__[key] == expected_region_values[key]
+            actual = r.__dict__[key]
```

Comment:
Could: Might be overkill but we use https://pypi.org/project/deepdiff/ to diff dictionaries in some places in `mx-bluesky` it does it entirely recursively, which is useful if the dict is complicated